Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalIsoTrk_Output_cff.py

```diff
@@ -11,14 +11,17 @@
     outputCommands = cms.untracked.vstring( 
         'keep *_IsoProd_*_*',
         'keep *_TkAlIsoProd_*_*',
+        'keep *_hbhereco_*_*',
```

Comment:
@bsunanda how much more data do we expect from this change?

@ghellwig: would it be possible to check using wf 1000.0 the expected output file change?